### DISCUSSION TITLE
Capa10x10 & Capa12x12

### DISCUSSION THREAD

**Discussion by numlockkey** [fairy-stockfish/Fairy-Stockfish#696] [Variants]
HI,

I just made this variants.ini file. When I load it in https://fairyground.vercel.app/, Capa10x10 appears in the list, but Capa12x12 does not. What am I doing wrong?  

Also, I want to give pawns the ability to jump 3 squares from their initial position in the Capa10x10 variant, and 4 squares in Capa12x12. How do I do that?

[capa10x10:capablanca]
maxRank=10
maxFile=10
startFen = rnbaqkcbnr/pppppppppp/10/10/10/10/10/10/PPPPPPPPPP/RNBAQKCBNR w KQkq - 0 1
archbishop = a
chancellor = c

[capa12x12:capablanca]
maxRank=12
maxFile=12
startFen = rnbacqkcabnr/pppppppppppp/12/12/12/12/12/12/12/12/PPPPPPPPPPPP/RNBACQKCABNR w KQkq - 0 1
archbishop = a
chancellor = c

**Comment by ianfab** [comment]
Fairy-SF only supports 12x10 boards as a maximum.

**Reply by numlockkey** [reply]
Is there a technical reason why it can't support 12x12?

Anyway, I changed the .ini file to 12x10. This time it shows up in the list, but when I select it nothing happens.

[capa12x10:capablanca]
maxRank=10
maxFile=12
startFen = rnbacqkcabnr/pppppppppppp/10/10/10/10/10/10/PPPPPPPPPPPP/RNBACQKCABNR w KQkq - 0 1

**Reply by ianfab** [reply]
Please make sure your FEN is valid, e.g., the 10s definitely should be 12s.

**Comment by numlockkey** [comment]
Here is one more problem:

I created Capa10x10.ini file with a custom pawn capable of moving up to 3 squares from the initial position.  It works, but only for the white side. The black side can only move by 1 square at first, but then, strangely, on the third move it all of sudden can move by 3 squares.

Here is the code:

[capa10x10:capablanca]
customPiece1 = p:fmWfceFifmnW3
pawnTypes = p
maxRank=10
maxFile=10
startFen = rnbaqkcbnr/pppppppppp/10/10/10/10/10/10/PPPPPPPPPP/RNBAQKCBNR w KQkq - 0 1

I'm also including screenshots of fairyground.vercel.app: 

![white3](https://github.com/fairy-stockfish/Fairy-Stockfish/assets/56490338/b279bd65-b969-4241-b368-1466763b969c)
![black1](https://github.com/fairy-stockfish/Fairy-Stockfish/assets/56490338/784919c8-30f4-421b-b96b-1b464eae2091)
![black3](https://github.com/fairy-stockfish/Fairy-Stockfish/assets/56490338/c62e9f62-290f-49d5-8846-23c65afe8f9f)

**Reply by ianfab** [reply]
You need to configure the doubleStep- and tripleStepRegion if you want them to differ from the default.
